### ISSUE TITLE
GIFs are always decoded as animated GIFs, so static GIFs cannot be rounded using BITMAP_ONLY rounding method. 

### ISSUE THREAD

**Issue by EzimetYusup** [facebook/fresco#188]
Hi!
I got a problem when roounding this image  http://bbs.bagdax.cn/uc_server/data/avatar/000/01/13/26_avatar_middle.jpg

Thanks

**Comment by tyronen** [comment]
What was the problem?

Was the link you posted the original image, or what it looked like with Fresco? Please post both so we can have an idea. Also please tell us the XML or code you used to round the image.

**Comment by EzimetYusup** [comment]
my XML file like this, 
```xml
   <com.facebook.drawee.view.SimpleDraweeView
            android:id="@+id/iv_forum_avatar"
            android:layout_centerHorizontal="true"
            android:layout_width="@dimen/width_forum_item_avatar"
            android:layout_height="@dimen/height_forum_item_avatar"
            fresco:roundAsCircle="true"
            fresco:roundingBorderWidth="2dp"
            fresco:roundingBorderColor="@color/item_forum_avatar_border"
            />
```
 a list of images are rounded fine but  http://bbs.bagdax.cn/uc_server/data/avatar/000/01/13/26_avatar_middle.jpg
this one is still not been rounded 
I call it from listview adapter  
 ```java
holder.iv_forum_avatar.setImageURI(Uri.parse(item.getAvatar()),context);
```

**Comment by tyronen** [comment]
Can you post a screenshot of what the image looks like in your app?

**Comment by EzimetYusup** [comment]
@tyronen  
![bagdax](https://cloud.githubusercontent.com/assets/7555642/7363332/865b742c-ed9a-11e4-94e7-0f56b97240d5.png)
android 5.1 google nexsus4

**Comment by stefanlam88** [comment]
hi EzimetYusup , u managed to solve? I m facing an issue where rounded image not working.
I follow your xml got display image but in rectangle style. You still got configure any?

**Comment by EzimetYusup** [comment]
Hi! @stefanlam88 , not yet ,I'm really busy these days , maybe @tyronen  is working on it .

**Comment by IanChilds** [comment]
It turns out that this image is actually a GIF, rather than a JPEG. 

We currently have a bug whereby we always decode GIFs as animated GIFs, even if they are not animated, and we don't round animated images when the rounding method is BITMAP_ONLY. I suggest perhaps using the rounding method OVERLAY_COLOR, which looks like it will work for you if you use the same overlay color as your background.

**Comment by tyronen** [comment]
Fixed in 0.5.1.
